### PR TITLE
Make FSGroup e2es [Feature:FSGroup], not [Skipped]

### LINKED ISSUE #19546: Reclassify [Skipped] tests: FSGroup
@pmorie there are a few tests that have been marked as skipped that need to be reenabled or otherwise taken care of (see #19505):

- "should provide podname as non-root with fsgroup [Conformance] [Skipped]": skipped in #16614; (also note we shouldn't have any tests that are both `[Conformance]` and `[Skipped]`)
- "when FSGroup is specified [Skipped]": "Remove [Skipped] when security context is enabled everywhere"

Perhaps we should mark these as `[Feature:FSGroup]` or something similar?

Happy to answer any questions if anything doesn't make sense.

cc @karlkfi

### PR DESCRIPTION
**Pull request by ikehz**
Fixes #19546, work toward #10548.

### CODE REVIEW

File: test/e2e/empty_dir.go [hunk 1]
```diff
@@ -37,8 +37,7 @@ var _ = Describe("EmptyDir volumes", func() {
 
 	f := NewFramework("emptydir")
 
-	// TODO: Remove [Skipped] when security context is enabled everywhere
-	Context("when FSGroup is specified [Skipped]", func() {
+	Context("when FSGroup is specified [Feature:FSGroup]", func() {
```
Comment (ixdy) [inline test/e2e/empty_dir.go]: the specs in here will be tagged with both [Feature] and [Conformance] - is that intended? seems to be at odds with the docs.

**Comment by thockin** [comment]
I honestly have no idea what these tags mean, so I trust you :)

**Comment by pmorie** [comment]
Personally, I think FSGroup tests should be conformance.  They are no more platform independent than any other e2e test in conformance IMO.

**Comment by pmorie** [comment]
I am fine with grouping them under feature:fsgroup in addition to conformance; I like that and have been wishing for feature tags like that.

**Comment by swagiaal** [comment]
Hmm.. shouldn't these be labeled [Feature:SecurityContext] like #19833 that is the reason they are "[skipped]"

**Comment by ihmccreery** [comment]
> Personally, I think FSGroup tests should be conformance. They are no more platform dependent than any other e2e test in conformance IMO.

@pmorie My impression has been that these tests require some non-default settings in order to work properly; if that's not the case (and also not the case for "Security Context" tests labeled in #19833) then they should just run as part of our core suite.

See https://github.com/kubernetes/kubernetes/blob/master/docs/devel/e2e-tests.md#kinds-of-tests for info (@thockin if you're curious).

**Comment by ihmccreery** [comment]
Ping @pmorie on above comment.

**Comment by pmorie** [comment]
@ihmccreery I am actually pretty sure now that I did not remember some of the particulars on this issue - going to remove LGTM for now, and we can talk about it tomorrow.  I believe there's going to be a hangup with FSGroup in GCE/GKE until the docker version picks up a patch we had to make to fix a bug with Docker's supplemental groups handling.  Apologies for spinning on this.

**Comment by ihmccreery** [comment]
@pmorie PTAL; made `[Feature:FSGroup]`.

**Comment by pmorie** [comment]
@ihmccreery :+1:

**Comment by ihmccreery** [comment]
@k8s-bot test this again please, issue #20913

**Comment by ihmccreery** [comment]
@k8s-bot test this again please, issue #20916

**Pull request merged**